src/components/ContextBar.js

```diff
@@ -102,7 +106,6 @@ export const ContextBar = ({ setDeletingWorkspace, setCloningWorkspace, setShari
           }, [icon('ellipsis-v', { size: 24 })])
         ]),
         h(Clickable, {
-          'aria-label': 'Cloud env config',
```

Comment:
Aria label is unnecessary because the tooltip (which is being used as the label) will be read by the screen reader… I verified this behavior.

Note also that when it is necessary to add an aria label, it is good to use complete words so it makes more sense to the user.

Reply:
Appreciate the accessibility edification 😊 
How about the `aria-label`s in `Clickable`s above and below?

Reply:
It is necessary for "Workspace Menu" because there is no tooltip that is being used as a label.

"Terminal button" is problematic. Because a tooltip IS being used there as a label, the aria-label does not get read out, and  instead the tooltip is read out. So what you get is a bit confusing.

Reply:
> It is necessary for "Workspace Menu" because there is no tooltip that is being used as a label.

Is there a reason why we couldn't add a `useTooltipAsLabel: true` and update `tooltip: 'Menu'` and `tooltip: 'Workspace Menu'` and get rid of `aria-label`?

> "Terminal button" is problematic. Because a tooltip IS being used there as a label, the aria-label does not get read out, and instead the tooltip is read out. So what you get is a bit confusing.

Agreed. Does it make sense to remove `useTooltipAsLabel: true` for that one?

Reply:
Sure, I made those changes. "Workspace Menu" is more useful than "Menu" for a tooltip anyway!

Reply:
Much appreciated! To be clear, I wasn't asking you to make the changes since your PR didn't introduce any of that. I was just trying to make sure I understand what was desirable as far as a11y 😊

Reply:
@kyuksel Sure, this was easy to change. But actually the ContextBar has far larger a11y issues, so I created https://broadworkbench.atlassian.net/browse/IA-3073 for it.